src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -357,8 +357,7 @@ private void InitializeLinuxLogging()
         public void Dispose()
         {
             // Not flushing the linux logger may lead to lost logs
-            int timeoutMilliseconds = 3000;
-            LinuxAppServiceLogger.Logger?.Stop(new TimeSpan(timeoutMilliseconds));
+            LinuxAppServiceLogger.Logger?.Stop(TimeSpan.FromSeconds(10));

```

Comment:
Is there a reason we transitioned from 3 to 10 seconds?

Reply:
I figured it wouldn't hurt to buy us a bit more time, especially during very noisy orchestrations. Ideally, we'd never hit the timeout anyways . I can revert back to 3 seconds if you feel strongly though :)

Reply:
I don't have any particularly strong concerns, was just curious.